bee-api/bee-rest-api/src/types/error.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_message::error::ValidationError as MessageError;
+
+use thiserror::Error;
+
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error("Invalid syntax for field \"{0}\"")]
```

Comment:
These error texts should not be capitalized according to this style rule for error messages that we never follow :trollface: